packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -485,121 +458,78 @@ describe('Checkout', () => {
+        describe('eventData ::', () => {
+            it('should return `isLoggedIn` and `serviceType` in an object`', () => {
                 // Arrange
                 const wrapper = shallowMount(VueCheckout, {
-                    store: createStore({
-                        ...defaultCheckoutState,
-                        isLoggedIn: true
-                    }),
+                    store: createStore(),
                     i18n,
                     localVue,
-                    propsData,
-                    data () {
-                        return {
-                            nonFulfillableError: {
-                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
-                                shouldShowInDialog: true
-                            }
-                        };
-                    }
+                    propsData
                 });
 
                 // Act
-                const result = wrapper.vm.shouldShowErrorDialog;
+                const result = wrapper.vm.eventData;
 
                 // Assert
-                expect(result).toBe(true);
+                expect(result).toEqual({ isLoggedIn: defaultCheckoutState.isLoggedIn, serviceType: defaultCheckoutState.serviceType });
             });
+        });
 
-            it('should return `false` if `nonFulfillableError.shouldShowInDialog` is `false`', () => {
+        describe('messageType ::', () => {
+            describe('when a message exists AND `shouldShowInDialog` is true', () => {
                 // Arrange
-                const wrapper = shallowMount(VueCheckout, {
-                    store: createStore({
-                        ...defaultCheckoutState,
-                        isLoggedIn: true
-                    }),
-                    i18n,
-                    localVue,
-                    propsData,
-                    data () {
-                        return {
-                            nonFulfillableError: {
-                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
-                                shouldShowInDialog: false
-                            }
-                        };
-                    }
-                });
-
-                // Act
-                const result = wrapper.vm.shouldShowErrorDialog;
-
-                // Assert
-                expect(result).toBe(false);
-            });
+                const dialog = {
+                    name: 'error-dialog'
+                };
 
-            it('should return `false` if there is no error', () => {
-                // Arrange
                 const wrapper = shallowMount(VueCheckout, {
                     store: createStore({
                         ...defaultCheckoutState,
-                        isLoggedIn: true
+                        message
                     }),
                     i18n,
                     localVue,
                     propsData
                 });
```

Comment:
The arrange and act sections should be inside the test, or at least in a `beforeEach`.